.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "typescript.validate.enable": false,
+    "typescript.suggest.includeAutomaticOptionalChainCompletions": false,
+    "javascript.validate.enable": false
+}
```

Comment:
Can you remove this from your commit? Perhaps add this file to .gitignore as well.